scripts/ci/check_compliance.py

```diff
@@ -542,6 +542,7 @@ def get_defined_syms(kconf):
     "SRAM2",  # Referenced in a comment in samples/application_development
     "STACK_SIZE",  # Used as an example in the Kconfig docs
     "STD_CPP",  # Referenced in CMake comment
+    "TAGOIO_HTTP_POST_LOG_LEVEL",  # Used as in samples/net/cloud/tagoio
```

Comment:
It is necessary to pass compliance tests.
```
./scripts/ci/check_compliance.py
Running checkpatch       tests in /home/gfbudke/zephyros/mainline/zephyr ...
Running Kconfig          tests in /home/gfbudke/zephyros/mainline/zephyr ...
Running Codeowners       tests in /home/gfbudke/zephyros/mainline/zephyr ...
Running Nits             tests in /home/gfbudke/zephyros/mainline/zephyr ...
Running Gitlint          tests in /home/gfbudke/zephyros/mainline/zephyr ...
Running pylint           tests in /home/gfbudke/zephyros/mainline/zephyr ...
Running Identity         tests in /home/gfbudke/zephyros/mainline/zephyr ...
1 checks failed
ERROR   : Test Kconfig failed: Found references to undefined Kconfig symbols. If any of these are false
positives, then add them to UNDEF_KCONFIG_WHITELIST in check_compliance.py in the ci-tools repo.

If the reference is for a comment like /* CONFIG_FOO_* */ (or
/* CONFIG_FOO_*_... */), then please use exactly that form (with the '*'). The
CI check knows not to flag it.

More generally, a reference followed by $, @, {, *, or ## will never be
flagged.

CONFIG_TAGOIO_HTTP_POST_LOG_LEVEL          samples/net/cloud/tagoio_http_post/src/main.c:8, samples/net/cloud/tagoio_http_post/src/sockets.c:8, samples/net/cloud/tagoio_http_post/src/wifi.c:8
```

Reply:
> Test Kconfig failed: Found references to undefined Kconfig symbols. If any of these are false
> positives, then add them to UNDEF_KCONFIG_WHITELIST in check_compliance.py in the ci-tools repo.

This is a bit weird because the CONFIG_TAGOIO_HTTP_POST_LOG_LEVEL is set in the sample Kconfig file so clearly it is not an undefined symbol.